org.freedesktop.LinuxAudio.Plugins.Calf.json

```diff
@@ -24,6 +24,7 @@
         "*.la"
     ],
     "modules": [
+        "shared-modules/linux-audio/jack2.json",
```

Comment:
no. jack header are in the 22.08 runtime. You shouldn't be using the shared module.

Reply:
yeah, doesn't build anyway
seems jack needs to be available at runtime or else plugins crash
what would be the way to go for this?
bundeling jack with the plugin seems... not appropriate

Reply:
the thing is the plugin shouldn't be requiring Jack anyway.